recipes/sphobjinv/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "sphobjinv" %}
+{% set version = "2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bskinn/sphobjinv/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
I'm guessing they are not publishing the sdist on PyPI, right? :weary:

Reply:
Right, not yet - may change in a future release.  I'm working with @bskinn on this item.